app/Support/Web3NftHandler.php

```diff
@@ -99,8 +111,8 @@ public function store(Collection $nfts, bool $dispatchJobs = false): void
             floor_price_retrieved_at = excluded.floor_price_retrieved_at,
             extra_attributes = excluded.extra_attributes,
             minted_block = excluded.minted_block,
-            minted_at = excluded.minted_at,
-            last_indexed_token_number = excluded.last_indexed_token_number
+            {$lastIndexedTokenConflict}
```

Comment:
I think we can simplify this entire diff to just

```sql
last_indexed_token_number = coalesce(excluded.last_indexed_token_number, collections.last_indexed_token_number)
```

Meaning that if we get a non-null timestamp we use it, otherwise we use the existing timestamp in the collections table which has the same effect as the above conditionals.

Reply:
applied it, test seems to like it as well and makes sense